src/app/Main/Body/QueryPage/Drawer/Drawer-Layout.component.tsx

```diff
@@ -68,7 +68,7 @@ export interface DrawerLayoutProps {
  * @returns {JSX.Element} - Drawer component
  */
 export function DrawerLayoutComponent(props: DrawerLayoutProps): JSX.Element {
-    const classes = useStyles((props.newWidth as unknown) as string);
+    const classes = useStyles((props.drawerWidth as unknown) as string);
```

Comment:
old story: useStyles(String(drawerWidth))

Reply:
Of cause, I tried this solution first. But in this case I get the error "Uncaught TypeError: rule is undefined" of the browser.

Reply:
Seems to be a problem with your input "drawerWidth". Please log the value to the console and check if it is always a number. I suspect that it is at some initial point undefined or null, then you should set it to a default for the useStyles.
Or you have to add a parameter to the useStyles function? => `useStyles = makeStyles(drawerWidth: string)`

Reply:
Done.

Reply:
I don't get it. I already have a default for the width. The width is always of type number. Even if I check the type before converting it doesn't work. I also tried to convert the type directly in the handler and not in the Layout. But that does not work either.

Reply:
Strange, you can leave it as it is because i have to test it myself to help further. Perhaps i will check it when you merged to develop.